### PR TITLE
Hannah, Project API

### PR DESCRIPTION
**Pull request by jesinkey**
https://express-api-hannahj.herokuapp.com/

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -1,34 +1,65 @@
...
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
```
Comment (KarolCislowski) [inline server.js:17]: Nice cleaning 🧹

File: server.js [hunk 2]
```diff
@@ -1,34 +1,65 @@
...
+app.get('/movies', (req, res) => {
+  const { director, year } = req.query;
+  let data = netflixData;
+  console.log(year);
+  if (director) {
+    data = movieByDirector(data, director);
+  }
+  if (year) {
+    data = movieByYear(data, year);
+  }
```
Comment (KarolCislowski) [inline server.js:38]: nice filtering 👍🏻

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Please send a Heroku link for testing 🤓

**Comment by jesinkey** [comment]
Sorry! https://express-api-hannahj.herokuapp.com/

**Review by KarolCislowski — APPROVED** [review]
Great API 👍🏻

**Pull request closed without merging**